README.rst

```diff
@@ -1,74 +1,219 @@
+2. You should activate certificates in the course from Studio in the
+   **Settings** > **Certificates**.
 
-(TODO: `Set up documentation <https://openedx.atlassian.net/wiki/spaces/DOC/pages/21627535/Publish+Documentation+on+Read+the+Docs>`_)
+Now, you can use the API. The API endpoints is protected with the same auth
+method as the Open edX platform. For generate a token, you can use the next
+endpoint:
 
-More Help
-=========
+- POST ``/<lms_host>/oauth2/access_token/``: Generate a token for the user. The
+  content type of the request must be ``application/x-www-form-urlencoded``.
 
-If you're having trouble, we have discussion forums at
-https://discuss.openedx.org where you can connect with others in the
-community.
+  **Body parameters**
 
-Our real-time conversations are on Slack. You can request a `Slack
-invitation`_, then join our `community Slack workspace`_.
+  - ``client_id``: Client ID of the OAuth2 application. You can find it in the
+    Django admin panel. Normally, it is ``login-service-client-id``.
+  - ``grant_type``: Grant type of the OAuth2 application. Normally, it is
+    ``password``.
+  - ``username``: Username of the user.
+  - ``password``: Password of the user.
+  - ``token_type``: Type of the token. By default, it is ``bearer``
 
-For anything non-trivial, the best path is to open an issue in this
+  **Response**
+
+  - ``access_token``: Access token of the user. You must use this token in the
+    ``Authorization`` header of the requests to the API.
+
+Finally, you are ready to use the API. The next endpoint is available:
+
+- GET ``/<lms_host>/platform-plugin-elm-credentials/<course_id>/api/credential-builder/``:
+  List all the topics in the course.
+
+  **Path parameters**
+
+  - ``course_id``: ID of the course.
+
+  **Query parameters**
+
+  - ``expired_at`` (optional): Date of the expiration of the credential. It
+    must be in the format ``YYYY-MM-DDTHH:mm:ss±hh:mm``. This parameter
+    modifies the properties ``validUntil`` and ``expirationDate`` of the ELMv3
+    format. By default, it is ``None``.
+  - ``to_file`` (optional): If it is ``true``, the response will be a JSON file
+    with the ELMv3 format. If it is ``false``, the response will be a JSON with
+    the ELMv3 format. By default, it is ``false``.
+
+
+Configuring plugin settings
+****************************
+
+Default settings are defined for **Unidigital project**. If you want to change
+the settings, you can do this in two ways:
+
+1. You can configure the settings in a specific course using ``Other Course Settings``
+   in the ``Advanced Settings`` section in Studio.
+
+   .. code-block:: json
+
+    {
+      ...
+      "ELM_CREDENTIALS_DEFAULTS": {
+        "primary_language_code": "POR",
+        "primary_language_map": {"fr": "FRA", "de": "DEU"},
+        "org_country_code": "PRT",
+        "issuer_id": "8fb2a434-e46b-4fd9-80d1-ce7ff11c0048"
+      }
+    }
```

Comment:
Please list which settings I can change, what each means, and what the Unidigital project defaults are.

Reply:
Done